.github/workflows/boards.yml

```diff
@@ -1,35 +1,86 @@
 name: Automate Boards
 on:
   issues:
-    types: [opened, labeled]
+    types: [opened, labeled, milestoned, demilestoned]
+  issue_comment:
+    types: [created]
 jobs:
-  # move to triaging board if new
+  # move to Triaging board if new
   triaging:
     runs-on: ubuntu-latest
     steps:
-      - uses: alex-page/github-project-automation-plus@v0.8.1
+      - uses: alex-page/github-project-automation-plus@v0
         if: github.event.action == 'opened' && !contains(github.event.issue.labels.*.name, 'backlog')
         with:
           action: update
           project: Triaging
           column: New
           repo-token: ${{ secrets.PROJECT_TOKEN }}
 
-  # move to backlog board if labeled as backlog
+  # remove [pending::feedback] if commented
+  pending_feedback:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: dessant/label-actions@v2
+        if: github.event_name == 'issue_comment' && github.event.action == 'created' && !github.event.issue.pull_request
+        with:
+          unlabel: pending::feedback
+          github-token: ${{ secrets.PROJECT_TOKEN }}
+
+  # move to Backlog board if labeled as [backlog]
   backlog:
     runs-on: ubuntu-latest
     steps:
-      - uses: alex-page/github-project-automation-plus@v0.8.1
+      - uses: alex-page/github-project-automation-plus@v0
         if: contains(github.event.issue.labels.*.name, 'backlog')
         with:
           action: delete
           project: Triaging
           column: Ready
           repo-token: ${{ secrets.PROJECT_TOKEN }}
-      - uses: alex-page/github-project-automation-plus@v0.8.1
+      - uses: alex-page/github-project-automation-plus@v0
         if: contains(github.event.issue.labels.*.name, 'backlog')
         with:
           action: update
           project: Backlog
           column: Unplanned
           repo-token: ${{ secrets.PROJECT_TOKEN }}
+
+  # remove [backlog] (& remove from Backlog board) if milestoned
+  milestoned:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: dessant/label-actions@v2
+        if: github.event.action == 'milestoned'
+        with:
+          unlabel: backlog
+          github-token: ${{ secrets.PROJECT_TOKEN }}
+      - uses: alex-page/github-project-automation-plus@v0
+        with:
+          action: delete
+          project: Backlog
```

Comment:
Should we keep those names as env vars or something to reduce the risk of breaking when we update the workflows and forget a line?

Reply:
Like the project names? I'd lean more towards verbosity in this case. The only benefit I see is just code reuse for it and then we'd obscure the name from the actual value being used when reviewing.

I won't die on this hill, but I'd vote to keep it here.

Reply:
I started going down this path and I have 2 concerns:
1. the variable names end up being very descriptive of its value which I'd personally feel inclined to update if any of the names were modified in the future:
   ```yaml
   env:
     BOARD_TRIAGING: Triaging
     STATE_READY: Ready
     LABEL_BACKLOG: backlog
     LABEL_FEEDBACK: pending::feedback
   ```
2. the one potentially nice win would be to store the token as an envvar which eliminates all of those duplicate `with` statements but idk if GHA is smart enough to keep those obfuscated once they become envvars?
   ```yaml
   env:
     INPUT_REPO_TOKEN: ${{ secrets.PROJECT_TOKEN }}
     INPUT_GITHUB_TOKEN: ${{ secrets.PROJECT_TOKEN }}
   ```